thunder/tests/test_update_aliases.py

```diff
@@ -70,6 +82,47 @@ def inplace_masked_fill_sample_generator(op, device, dtype, requires_grad, **kwa
         _inplace_opinfos.append(inplace_opinfo)
 
 
+@pytest.fixture
+def turn_off_tf32_and_set_seed(monkeypatch):
+    monkeypatch.setenv("NVIDIA_TF32_OVERRIDE", "0")
+    torch.manual_seed(42)
```

Comment:
Should these state changes be reverted later in the test? Otherwise, IIUC tests which run after this test will see these state updates.